packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -73,23 +74,21 @@ class FloatingActionButton extends StatelessWidget {
     this.foregroundColor,
     this.backgroundColor,
     this.heroTag = const _DefaultHeroTag(),
-    this.elevation = 6.0,
-    this.highlightElevation = 12.0,
-    double disabledElevation,
+    this.elevation,
+    this.highlightElevation,
+    this.disabledElevation,
     @required this.onPressed,
     this.mini = false,
-    this.shape = const CircleBorder(),
+    this.shape,
     this.clipBehavior = Clip.none,
     this.materialTapTargetSize,
     this.isExtended = false,
-  }) : assert(elevation != null && elevation >= 0.0),
-       assert(highlightElevation != null && highlightElevation >= 0.0),
+  }) : assert(elevation == null || elevation >= 0.0),
```

Comment:
Update the documentation above now that these 2 may be null

Reply:
Done